streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/watcher/FlinkAppHttpWatcher.java

```diff
@@ -504,6 +503,26 @@ private void handleNotRunState(
                 doPersistMetrics(application, true);
                 cleanOptioning(optionState, application.getId());
                 break;
+            case FINISHED:
+                log.info(
+                    "[StreamPark][FlinkAppHttpWatcher] getFromFlinkRestApi, job state {}, stop tracking and delete stopFrom!",
+                    currentState.name());
+                cleanSavepoint(application);
+                application.setState(currentState.getValue());
+                if (StopFromEnum.NONE.equals(stopFrom) || applicationInfoService.checkAlter(application)) {
+                    if (StopFromEnum.NONE.equals(stopFrom)) {
+                        log.info(
+                            "[StreamPark][FlinkAppHttpWatcher] getFromFlinkRestApi, job finished is not form StreamPark,savepoint expired!");
+                        savepointService.expire(application.getId());
+                    }
+                    stopCanceledJob(application.getId());
+                    // 不发送报警，因为FINISHED对于批作业是正常状态
+                    // doAlert(application, FlinkAppStateEnum.CANCELED);
```

Comment:
- What about dropping the commented line if we not need it ?
- For the current patch, We'd better to follow [1] to create an issue for tracking the change of the current PR.
- ditto to the title of the PR and commit message.

[1] https://streampark.apache.org/community/submit_guide/code_style_and_quality_guide#1-pull-requests--changes-rule